### PR TITLE
community:(baidu_qianfan)[patch]: standardize init args 

### PR DESCRIPTION
**Pull request by liugddx**
Related to https://github.com/langchain-ai/langchain/issues/20085

@baskaryan

### CODE REVIEW

File: libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ class QianfanChatEndpoint(BaseChatModel):
     streaming: Optional[bool] = False
     """Whether to stream the results or not."""
 
-    request_timeout: Optional[int] = 60
+    timeout: Optional[int] = 60
```
Comment (baskaryan) [inline libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py:106]: need to make sure we do this in a backwards compatible way:
```python
request_timeout: Optional[int] = Field(default=60, alias="timeout")
```
see https://github.com/langchain-ai/langchain/pull/20161 for an example

and we should add a unit test to make sure current and new init args work

Comment (liugddx) [reply libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py:106]: > need to make sure we do this in a backwards compatible way:
> 
> ```python
> request_timeout: Optional[int] = Field(default=60, alias="timeout")
> ```
> 
> see #20161 for an example
> 
> and we should add a unit test to make sure current and new init args work

I see.Thanks for review.

**Review by baskaryan — COMMENTED** [review]
need to add `class Config`

**Comment by liugddx** [comment]
@baskaryan ptal

**Pull request merged**